pyroengine/engine/engine.py

```diff
@@ -90,6 +90,8 @@ def predict(self, frame, pi_zero_id=None):
                 self.event_appening[pi_zero_id] = True
 
             if self.use_api and self.event_appening[pi_zero_id]:
+                # Reduce image size to save bandwidth
+                frame = frame.resize(((960, 720)))
```

Comment:
Perhaps we should set the compressed size as a constructor argument (and its class attribute counterpart) So that if we change this in the future, we only have to pass another size when instantiating the engine